src/App.tsx

```diff
@@ -1,9 +1,36 @@
+import { Routes, Route, Navigate } from 'react-router-dom';
 import './App.scss';
+import 'bulma/css/bulma.min.css';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+import { Header } from './Components/Header';
+import { PeopleTable } from './Components/PeopleTable';
+import { HomePage } from './Components/HomePage';
+import { Page404 } from './Components/Page404';
 
-export default App;
+export const App: React.FC = () => {
+  return (
+    <div className="App">
+      <div className="container is-max-desktop">
+        <Header />
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+
+          <Route
+            path="people"
+            element={(
+              <div>
+                <h1 className="title has-text-centered notification is-primary">
+                  People Table
+                </h1>
+                <PeopleTable />
+              </div>
+            )}
+          />
+
+          <Route path="*" element={<Page404 />} />
+          <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
`replace`?
here is why – https://reach.tech/router/api/navigate
also – this one
![Screen Shot 2022-08-05 at 09 31 42](https://user-images.githubusercontent.com/75496546/183015874-4febb8dc-7972-4e73-a320-65a2a52a4aca.png)